docs/api/extending_the_rest_api.md

```diff
@@ -60,6 +60,17 @@ my_rest_hello_world:
     methods: [GET]
 ```
 
+To configure whether the endpoint uses CSRF validation, add the `csrf_protection` setting to the route:
+
+``` yaml
+my_rest_hello_world:
+    path: '/my_rest_bundle/hello/{name}'
+    defaults:
+        _controller: App\Rest\Controller\DefaultController::sayHello
+        csrf_protection: false
+    methods: [GET]
+```
+
```

Comment:
Please just make sure documentation mention that CSRF protection is enabled by default for all POST, PUT, DELETE requests

Reply:
```suggestion

CSRF protection is enabled by default for all POST, PUT, DELETE requests.

```